pkg/trace/watchdog/logonpanic_test.go

```diff
@@ -24,11 +24,7 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
-	err = seelog.ReplaceLogger(logger)
-	if err != nil {
-		panic(err)
-	}
-	log.SetupLogger(logger, "INFO")
+	log.SetLogger(logger)
```

Comment:
Would be nice to get rid of `seelog` entirely by using a mock logger here instead (maybe not very important, it just would be nice to not depend on it)